ui/src/components/SampleControls/ZoomControl.jsx

```diff
@@ -11,10 +11,12 @@ const ZOOM_HWO_ID = 'diffractometer.zoom';
 function ZoomControl() {
   const dispatch = useDispatch();
 
-  const { state, value, commands } = useSelector(
+  const zoomObj = useSelector(
     (state) => state.beamline.hardwareObjects[ZOOM_HWO_ID],
   );
 
+  const { state, value, commands } = zoomObj;
```

Comment:
Note that the `state` variable is now declared after, so it's fine. It wouldn't have worked with the old `var` keyword.